codice/alliance
# CAL-143 Incorporate Catalog UI CSS Styling into Image Chipping Application

#### What does this PR do?
 - Applies minimal CSS styling to the Alliance image chipping app, for consistency with the Catalog UI. 
 - Added header to provide some instruction to the user on how to use the app. 
 - Disabled the `Download` button until a bounding box selection is made, to prevent a server error.
 - Changed the names of the buttons to be more descriptive.
 - Added custom cursor when user is drawing the bounding box.  

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@andrewkfiedler @bdeining @codice/ui 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@pklinef

#### How should this be tested?
1. Build and run Alliance.
2. Ingest an image NITF file.
3. Choose `Chip Image` action provider on ingested NITF.
4. Verify the `Download` and `Reset` buttons are consistent with Catalog UI font/color styling.
5. Regression test that image chip can be selected and downloaded from both the action provider popup and the new window.

#### Any background context you want to provide?
Until the Catalog UI is merged into DDF, the styles from the Catalog UI CSS have been manually applied to the image chipping application CSS. Additional UX enhancements have been recommended and are beyond the scope of this ticket.

#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-143

#### Screenshots (if appropriate)
![screen shot 2016-09-26 at 9 50 53 am](https://cloud.githubusercontent.com/assets/16792154/18841385/c9015884-83ce-11e6-85b2-f23a81c6dd1a.png)

#### Checklist:
